packages/db-base/lib/tools.js

```diff
@@ -10,7 +10,7 @@ module.exports.getControllerDir = () => controllerDir;
  * @param {((error: Error | null | undefined, ...args: any[]) => void) | null | undefined} callback - callback function to be executed
  * @param {Error | string | null | undefined} error - error which will be used by the callback function. If callback is not a function and
  * error is given, a rejected Promise is returned. If error is given but it is not an instance of Error, it is converted into one.
- * @param {any[]} args - as many arguments as needed, which will be returned by the callback function or by the Promise
+ * @param {...any} args - as many arguments as needed, which will be returned by the callback function or by the Promise
```

Comment:
From a typescript perspective, this was correct before.

Reply:
IDE was complaining. JSdoc mentions it this way
https://jsdoc.app/tags-param.html see under `Allows a parameter to be repeated`

But yes.. this will be ts soon too I guess

Reply:
Hm, VSCode supports both `...any` and `any[]` it seems.